src/pages/2018-11-21-bull/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Processing jobs across microservices using Bull 🐂
+date: '2018-11-21T22:12:03.284Z'
+path: '/job-processor-async/'
+---
+
+> An in-depth explanation into why we invested in the open source package, <a href="https://github.com/OptimalBits/bull" target="_blank">Bull</a>, even though it didn't quite fit our needs.
+
+Into the microservice world we go, exploring where no developer has gone before. When switching to a service oriented stack versus the ol' monolith, new problems arise. The simple async job processor of the past doesn't fit in this new architecture. We found <a href="https://github.com/OptimalBits/bull" target="_blank">Bull</a>, a Node.js package, to cover our needs, but needed tweaks to work in this new world.
```

Comment:
It may help to frame this post in the context of a specific job processing example, especially to help those unfamiliar with the fundamentals of async job queueing, and the types of problems it solves.